common/jmeter/saml/SAML2_SSO_Redirect_Binding.jmx

```diff
@@ -309,6 +319,16 @@ sessionDataKey=([a-z0-9-]+)\&amp;?</stringProp>
             <stringProp name="Sample.scope">all</stringProp>
           </RegexExtractor>
           <hashTree/>
+          <RegexExtractor guiclass="RegexExtractorGui" testclass="RegexExtractor" testname="Redirect URL Extractor" enabled="true">
```

Comment:
why we need this change ?

Reply:
Removed by https://github.com/wso2/performance-is/pull/128/commits/ef88a709e62a267d93e2cf768f66851ed0a8035e